src/OpenSpaceToolkit/Physics/Environment/Magnetic/Earth.cpp

```diff
@@ -145,7 +152,7 @@ MagneticModel* Earth::Impl::MagneticModelFromType(const Earth::Type& aType, cons
 
     using ostk::physics::environment::magnetic::earth::Manager;
 
-    if (aType == Earth::Type::Dipole)
+    if (aType == Earth::Type::Undefined || aType == Earth::Type::Dipole)
```

Comment:
Let's add a TBI here for the Dipole so that we don't confuse why it's returning a nullptr later?